src/status_im/accounts/update/core.cljs

```diff
@@ -25,7 +25,7 @@
     (let [updated-chat  (assoc chat :resend? "contact-update")
           tx            [(transport-store/save-transport-tx {:chat-id chat-id
                                                              :chat    updated-chat})]
-          success-event [:transport/contact-message-sent chat-id]]
+          success-event [:transport/contact-message-sent chat-id nil]]
```

Comment:
why do you need to add nil here? Isn't it going to be nil anyway if you don't provide it?

Reply:
not sure, I could check, but i'd rather be explicit, there's no harm and it's clearer imo